kext/FileNVRAM/FileNVRAM.cpp

```diff
@@ -217,7 +217,7 @@ void FileNVRAM::copyUnserialzedData(const char* prefix, OSDictionary* dict)
             
             if(prefix)
             {
-                size_t size = strlen(prefix) + sizeof(NVRAM_SEPERATOR) + strlen(key->getCStringNoCopy());
+                size_t size = strlen(prefix) + sizeof(NVRAM_SEPERATOR) + strlen(key->getCStringNoCopy()) +1;
```

Comment:
OK, I'll take a look at the screenshots.
I'd suggest also testing with the line changed to the following instead:
size_t size = strlen(prefix) + strlen(NVRAM_SEPERATOR) + strlen(key->getCStringNoCopy()) +1;

This should be equivalent to the original line, and if it also shows an issue, then the bug you're trying to fix is caused by something else.